src/lib/wikitools/Image.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+  import { themeId } from '$lib/pickers/stores';
+
+  export let lightsrc: string;
+  export let darksrc: string;
+  export let alt: string;
+
+  let prefersDark: boolean = window.matchMedia(
```

Comment:
This variable should probably be declared reactively (`$:`) but I'm not sure if that's possible in this case.

Reply:
@Venrix Also consider that `window.matchMedia` might crash the page if it's run server side.